src/hooks/useUniswapXSwapCallback.ts

```diff
@@ -117,6 +119,10 @@ export function useUniswapXSwapCallback({
             timeToSignSinceRequestMs: Date.now() - beforeSign,
           }),
           ...analyticsContext,
+          // TODO (WEB-2993): remove these.
```

Comment:
can u also write in the comment why ur temporarily adding them in the first place? can just be like `// remove these after testing

Reply:
done